test/test_torch.py

```diff
@@ -1851,6 +1872,10 @@ def test_div(self):
             res2[i, 3] = res2[i, 3] / 2
         self.assertEqual(res1, res2)
 
+        a1 = torch.tensor([4.2, 6.2], dtype=torch.bfloat16)
+        a2 = torch.tensor([2., 2.], dtype=torch.bfloat16)
+        self.assertEqual(a1 / a2, torch.tensor([2.1, 3.1], dtype=torch.bfloat16), 0.1)
```

Comment:
same here.

Reply:
no, because this test doesnt test all dtypes